educator/tests/Test/Dscp/DB/SQLite/Queries.hs

```diff
@@ -302,7 +302,7 @@ spec_Instances = do
                         ptId <- DB.createTransaction trans
                         return ptId
 
-                    DB.createBlock Nothing
+                    _ <- runMaybeT $ DB.createPrivateBlock Nothing
```

Comment:
"Fail silently" is not the best behaviour, I believe.

Reply:
Right, makes sense